src/main/java/edition/academy/seventh/display/BookFilter.java

```diff
@@ -0,0 +1,62 @@
+package edition.academy.seventh.display;
+
+/**
+ * Available {@link Pagination} book filters. Each {@code Enum constant} implements value
+ * initialized with native SQL query adequates to the name meaning.
+ *
```

Comment:
Ja bym dał taki javadoc:
Native SQL queries for {@ Pagination}

Reply:
Nie jest to tylko typ wyliczeniowy pod zapytania więc taki opis wprowadzałby w błąd.